.eslintrc.json

```diff
@@ -4,10 +4,12 @@
     "plugin:cypress/recommended"
   ],
   "plugins": [
+    "babel",
```

Comment:
At some point, these changes should be rolled back into https://github.com/mattermost/eslint-plugin-mattermost so that they can be used in mattermost-redux and the mobile app

Reply:
I agree 👍. I'll follow up with a sync PR (migrate relevant to `eslint-plugin-mattermost`). I was thinking to add there directly, but didn't want conflict/errors in-between merges.